reference/enterprise-api-ref/cmdb-api.markdown

```diff
@@ -0,0 +1,317 @@
+  Number of results to skip for the processed
+  query. The Mission Portal uses this for pagination. Optional parameter.
+* **limit**  *(integer)*
+  Limit the number of results in the query.
+* **hostContextInclude** *(array)*
+  Includes only results that concern hosts which have all specified CFEngine contexts (class) set. Optional parameter.
+* **hostContextExclude** *(array)*
+  Excludes results that concern hosts which have specified CFEngine context (class) set. Hosts that have at least one of the specified contexts set will be excluded from the results. Optional parameter.
+
+    
+**Example request (curl):**
+ 
+```
+curl -k --user <username>:<password> \
+  -X GET \
+  https://hub.cfengine.com/api/cmdb?epochFrom=2&hostContextInclude[]=ubuntu \
+  -H 'content-type: application/json' 
+```
+
+**Example response:**
+
+```
+HTTP 200 Ok
+{
+    "data": {
+        "SHA=fdab725e8fb18aa0ad194477be8a2a1338b4b29f6a8597819af89697e432418f": {
+            "classes": {
+                "My_class": {},
+                "My_class2": {
+                    "comment": "comment helps to understand what this class serves for"
+                }
+            },
+            "variables": {
+                "HubCMDB:My.hostname": {
+                    "value": "host1.cfengine.com",
+                    "comment": "comment"
+                },
+                "Namespace:BundleName.VariableName": {
+                    "value": "myvalue"
+                }
+            }
+        }
+    },
+    "meta": {
+        "count": "1",
+        "page": 1,
+        "timestamp": 1619116399,
+        "total": "1",
+        "most_recent_cmdb_change": "13"
+    }
+}
+```
+
+## Get configuration by hostkey
+
+**URI:** https://hub.cfengine.com/cmdb/:hostkey
+
+**Method:** GET
+
+**Parameters:**
+* **hostkey** *(string)*
+  Unique host identifier.
+
+**Example request (curl):**
+
+```
+curl -k --user <username>:<password> \
+  -X GET \
+  https://hub.cfengine.com/api/cmdb/SHA=f622992fa4525070f47da086041a38733496f03a77880f70b1ce6784c38f79ab 
+```
+
+**Example response:**
+
+```
+HTTP 200 Ok
+{
+    "hostkey": "SHA=f622992fa4525070f47da086041a38733496f03a77880f70b1ce6784c38f79ab",
+    "value": {
+        "classes": {
+            "My_class": {},
+            "My_class2": {
+                "comment": ""
+            }
+        },
+        "variables": {
+            "HubCMDB:My.hostname": {
+                "value": "host1.cfengine.com",
+                "comment": "My hostname should be set to this"
+            },
+            "Namespace:BundleName.VariableName": {
+                "value": "myvalue"
+            }
+        }
+    }
+}
+```
+
+## Create new configuration
+
+**URI:** https://hub.cfengine.com/cmdb
+
+**Method:** POST
+
+**Parameters:**
+* **hostkey** *(string)*
+  Unique host identifier.
+* **classes** *(JSON object)*
+  The format is a JSON object where the key is class name and value is another JSON object
+  with an optional `comment` property.
```

Comment:
So `tags` are not supported by the API? (and the MP side of CMDB in general?)

Reply:
supported, added here